featuretools/primitives/__init__.py

```diff
@@ -1,10 +1,19 @@
+# flake8: noqa
```

Comment:
should be able to delete this

Reply:
Can't remove it here because this line `from featuretools.primitives.standard import *` causes a lint failure